### PR TITLE
Rewrite of requirements and more emphasis on bare metal

### PR DESCRIPTION
**Pull request by stakeworld**
Tried to make the structure a little bit more logical and added a little more emphasis on bare metal compared to VPS.

P.S. is the split between kusama and polkadot for instructions about setting up a validator not a little double work? Now the two documents deviate because changes are made in one or the other. In my opinion more logical would be a page about setting up a validator (the polkadot binary, hardware, installing, building etc) and some information about polkadot/kusama specifics, like "How Many KSM/DOT Do I Need"

### CODE REVIEW

File: docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md [hunk 1]
```diff
@@ -61,22 +61,17 @@ instead.
 
 ### Requirements
 
-The most common way for a beginner to run a validator is on a cloud server running Linux. You may
-choose whatever [VPS](#vps-list) provider that your prefer, and whatever operating system you are
-comfortable with. For this guide we will be using **Ubuntu 18.04**, but the instructions should be
-similar for other platforms.
+The most common way for a beginner to run a validator is on a cloud/VPS server. Another way is to install on a dedicated server ("bare metal"). You may choose whatever [VPS](#vps-list) or [dedicated server](#dedicated-server-providers) provider you prefer but keep in mind that node hardware should run at a level equal to or above the standard benchmark machines.
```
Comment (profradha) [inline docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md]:
```suggestion
The most common way for a beginner to run a validator is on a cloud/VPS server. Another way is to install on a dedicated server ("bare metal"). You may choose any [VPS](#vps-list) or [dedicated server](#dedicated-server-providers) provider you prefer, but keep in mind that node hardware should run at a level equal to or above the standard benchmark machines. 
```

File: docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md [hunk 2]
```diff
@@ -61,22 +61,17 @@ instead.
...
-choose whatever [VPS](#vps-list) provider that your prefer, and whatever operating system you are
-comfortable with. For this guide we will be using **Ubuntu 18.04**, but the instructions should be
-similar for other platforms.
+The most common way for a beginner to run a validator is on a cloud/VPS server. Another way is to install on a dedicated server ("bare metal"). You may choose whatever [VPS](#vps-list) or [dedicated server](#dedicated-server-providers) provider you prefer but keep in mind that node hardware should run at a level equal to or above the standard benchmark machines. 
 
-Node hardware should run at a level at or above the standard benchmark machines. Many VPS providers
-seem to fall short in practice, despite good looking specifications. Most validators who are
-serious about setting up an active validator progress to a
-[dedicated server](#dedicated-server-providers).
+Many VPS providers seem to fall short in practice, despite good looking specifications. Most validators who are actively validating progress to a dedicated server.
```
Comment (profradha) [inline docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md]: retain the markdown link to dedicated server providers section

```suggestion
Many VPS providers seem to fall short in practice, despite good looking specifications. Most validators who are
serious about setting up an active validator progress to a
[dedicated server](#dedicated-server-providers).
```

File: docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md [hunk 3]
```diff
@@ -61,22 +61,17 @@ instead.
...
-seem to fall short in practice, despite good looking specifications. Most validators who are
-serious about setting up an active validator progress to a
-[dedicated server](#dedicated-server-providers).
+Many VPS providers seem to fall short in practice, despite good looking specifications. Most validators who are actively validating progress to a dedicated server. 
 
-The transactions weights in Polkadot were benchmarked on standard hardware. It is recommended that
-validators run at least the standard hardware in order to ensure they are able to process all blocks
-in time. The following are not _minimum requirements_ but if you decide to run with less than this
-beware that you might have performance issue.
+#### Software
```
Comment (profradha) [inline docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md]:
```suggestion
#### Operating System
```

File: docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md [hunk 4]
```diff
@@ -61,22 +61,17 @@ instead.
...
 
-#### Standard Hardware
+You may use whatever operating system you are comfortable with. For this guide we will be using **Ubuntu 20.04**, but the instructions should be similar for other platforms.
+
+#### Hardware
+
+The transactions weights in Polkadot were benchmarked on standard hardware. It is recommended that validators run at least the standard hardware in order to ensure they are able to process all blocks in time. The following are not _minimum requirements_ but if you decide to run with less than this beware that you might have performance issue.
```
Comment (profradha) [inline docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md]: rephrasing a little bit

```suggestion
The following specs are not the _minimum requirements_, but if you decide to run with less than this, your validator nodes might face performance issues. The transaction weights on Kusama are benchmarked on standard hardware. It is recommended that the validator nodes run on machines matching the standard hardware specs in order to ensure block processing and production in time. 
```

**Review by DrW3RK — CHANGES_REQUESTED** [review]
Some minor suggestions

**Comment by DrW3RK** [comment]
> P.S. is the split between kusama and polkadot for instructions about setting up a validator not a little double work?

You're right. We will explore the possibility of using the same document to conditionally render for both Kusama and Polkadot.

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**